pkgs/servers/mail/mailman/web.nix

```
@@ -15,6 +15,13 @@ buildPythonPackage rec {
     leaveDotGit = true;
   };
 
+  patches = [
+    (fetchpatch {
```

do you mind adding a comment about what the intent of the patch is doing, and when the patch is likely not to be needed
```suggestion
  patches = [
    # does x, y, z. Should be able to be removed after >=X.Y.Z
    (fetchpatch {
```